templates/tabletop-development.ini.erb

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+[secret]
```

Comment:
At present, ``[secret]`` is completely ignored by the ``c.include_plugin_config`` method, so having this section in a plugin won't affect anything adversely.  In the long run, we should probably extend ``include_plugin_config`` to merge secret settings together, though I was just ambivalent enough about that to not implement that until I was sure it was a good idea.